apps/content/app/pom.xml

```diff
@@ -0,0 +1,119 @@
+    </parent>
+
+    <artifactId>content-apps</artifactId>
+    <packaging>content-package</packaging>
+
+    <name>${module.name} - App Package</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>filevault-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <packageType>application</packageType>
+                    <properties>
+                        <cloudManagerTarget>none</cloudManagerTarget>
+                    </properties>
+                    <filters>
+                        <filter><root>/apps/dx/components/content/marketo</root></filter>
+                    </filters>
+                    <embeddeds>
+                        <embedded>
+                            <groupId>com.adobe.dx</groupId>
+                            <artifactId>com.adobe.dx.content.core</artifactId>
+                            <target>/apps/dx/install</target>
+                            <filter>true</filter>
+                        </embedded>
+                        <embedded>
+                            <groupId>com.adobe.dx</groupId>
+                            <artifactId>content-content</artifactId>
+                            <target>/apps/dx-packages/content/install</target>
+                            <filter>true</filter>
+                        </embedded>
+                    </embeddeds>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>com.day.jcr.vault</groupId>
+                <artifactId>content-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.sling</groupId>
+                <artifactId>htl-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
+        <profile>
+            <id>install-all</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.day.jcr.vault</groupId>
+                        <artifactId>content-package-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>install-package</id>
+                                <goals>
+                                    <goal>install</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.adobe.dx</groupId>
+            <artifactId>com.adobe.dx.content.core</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
```

Comment:
do you need to specify the scope? `compile` in that context sounds weird

Reply:
The `com.adobe.dx.content.core` bundle is not exposed in maven (skipped) and would rather be embedded inside app. Hence the compile scope. 
If you wish to expose the bundle then I can remove the scope.